src/styles/utils/_mixins.scss

```
@@ -0,0 +1,4 @@
+@mixin cell {
+  height: $cellSize;
+  width: $cellSize;
+}
```

```suggestion
@mixin cell($size) {
  height: $size;
  width: $size;
}
```

better make it more generic 😉

---

Received